recipes/socat/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "socat" %}
+{% set version = "1.7.3.2" %}
+{% set md5 = "607a24c15bd2cb54e9328bfbbd3a1ae9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.bz2
+  url: http://www.dest-unreach.org/{{ name }}/download/{{ name }}-{{ version }}.tar.bz2
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+  rpaths:
+    - lib/
+
+requirements:
+  build:
+    - toolchain
+    - openssl 1.0.2*
+    - readline 6.*
+  run:
+    - openssl 1.0.2*
+    - readline 6.*
+
+test:
+  commands:
+    - filan -h
+    - procan -h
+    - socat -h
```

Comment:
I made the changes, I am not sure why the conda inspect one was not picked up by the github ui.